### DISCUSSION TITLE
Do different helix process share the same lsp ?

### DISCUSSION THREAD

**Discussion by Dangerise** [helix-editor/helix#9500] [General] [answered]
As the title, can I open multiple helix editors in the same Rust crate and share the same lsp ? I'm afraid that multiple lsp will fill my RAM

**Comment by the-mikedavis** [comment, marked as answer]
No each Helix process starts its own language servers

**Reply by Dangerise** [reply]
Could it be a new feature?

**Reply by the-mikedavis** [reply]
See https://github.com/helix-editor/helix/issues/7535, it's covered by #312

**Comment by Dangerise** [comment]
Thanks
